test/e2e/network_test.go

```diff
@@ -46,3 +62,352 @@ func testHybridOverlayRunning(t *testing.T) {
 			vm.GetCredentials().GetInstanceId())
 	}
 }
+
+// testEastWestNetworking deploys Windows and Linux pods, and tests that the pods can communicate
+func testEastWestNetworking(t *testing.T) {
+	testCtx, err := NewTestContext(t)
+	require.NoError(t, err)
+
+	for _, vm := range gc.windowsVMs {
+		externalIP := vm.GetCredentials().GetIPAddress()
+		node, err := testCtx.getNode(externalIP)
+		affinity, err := getAffinityForNode(node)
+		require.NoError(t, err, "Could not get affinity for first node")
```

Comment:
We are following a bad pattern with capitalizing error messages here which I carried over from WSU. Let us avoid that here.